src/services/configService.js

```diff
@@ -69,4 +63,19 @@ export class ConfigService {
     return HttpClient.put(`${ENDPOINT}solution-settings/logo`, logo, options)
       .map(prepareLogoResponse);
   }
+
+  /* get solution settings */
+  static getSolutionSettings() {
+    return HttpClient.get(`${ENDPOINT}solution-settings/theme`)
+      .map(toSolutionSettingThemeModel)
+      .catch(error =>
```

Comment:
Why is this error handling different than the rest of the application?

Reply:
Because, When the application loads for the first time, there won't be any solution settings in the storage. It will throw a 404 not found error. But we need to ignore that and take static values defined in UI state. I can add a comment for this.